servo/connectors/olas/server_classes.py

```diff
@@ -0,0 +1,104 @@
+from typing import List, Dict
+from pydantic import BaseModel
+
+__all__ = [
+    "Message",
+    "Prediction",
+    "TimeSeries",
+    "PredictionResult",
+    "Id",
+    "PodModelWithId",
+    "TrainPodModel",
+    "Allocation",
+    "NodeType",
+    "NodeMetrics",
+    "PodMetrics",
+    "Metrics",
+    "GetMetrics",
+]
+
+
+class Message(BaseModel):
+    ts: float
+    msg: str
+
+
+class Prediction(BaseModel):
+    src: str
+
+
+class TimeSeries(BaseModel):
+    ts: float
+    value: float
+
+
+class PredictionResult(BaseModel):
+    value: float
+    err: str = ''
+
+
+class Id(BaseModel):
+    id: float
+
+
+class PodModelWithId(BaseModel):
+    model_id: float
+    model: str
+
+
+class TrainPodModel(BaseModel):
+    start: float
+    end: float
+
+
+class Allocation(BaseModel):
+    cpu_request: float = 0
+    cpu_limit: float = 0
+    memory_request: float = 0
+    memory_limit: float = 0
```

Comment:
`servo.connectors.kubernetes.CPU` and `servo.connectors.kubernetes.Memory` should cover this I think